lib/rules/shorthand-property-no-redundant-values/index.js

```diff
@@ -15,7 +14,14 @@ const messages = ruleMessages(ruleName, {
   rejected: (unexpected, expected) => `Unexpected longhand value '${unexpected}' instead of '${expected}'`,
 })
 
-const shorthandableProperties = new Set(Object.keys(shorthandData))
+const shorthandableProperties = new Set([
```

Comment:
Can you please add a comment explaining why these specific properties are checked, and not others?